packages/docs-app/src/examples/core-examples/sectionExample.tsx

```diff
@@ -134,8 +144,14 @@ export class SectionExample extends React.PureComponent<ExampleProps, SectionExa
         return (
             <Example options={options} {...this.props}>
                 <Section
+                    // A `key` is provided here to force the component to
+                    // re-mount when `defaultIsOpen` is changed, otherwise
+                    // the local state in the `Collapse` component is not
+                    // updated.
+                    key={String(defaultIsOpen)}
```

Comment:
Using `key` to reset state and animations is one of my favorite tricks in the pocket too. 🙂

Reply:
Haha yeah it's a neat little trick! :)